pkg/services/accesscontrol/permreg/permreg.go

```diff
@@ -94,6 +94,9 @@ func newPermissionRegistry() *permissionRegistry {
 		"roles":           "roles:uid:",
 		"services":        "services:",
 		"receivers":       "receivers:uid:",
+
+		"secrets-manager.securevalues": "secrets-manager.securevalues:name:",
+		"secrets-manager.keepers":      "secrets-manager.keepers:name:",
```

Comment:
@kalleep @gamab I still need these right?

Reply:
Yes you do. I think, for consistency with other resources, you should use `uid` (which is the k8s name)
```suggestion
		"secrets-manager.keepers":      "secrets-manager.keepers:uid:",
```